.travis.yml

```diff
@@ -13,34 +13,50 @@ env:
     - IOS_SDK="iphonesimulator10.3"
     - IOS10_DESTINATION="OS=10.3,name=iPhone 7"
 before_install:
-  - openssl aes-256-cbc -K $encrypted_566f29436663_key -iv $encrypted_566f29436663_iv -in Source/SupportingFiles/Credentials.swift.enc -out Source/SupportingFiles/Credentials.swift -d
+  - openssl aes-256-cbc -K $encrypted_a31ce544e168_key -iv $encrypted_a31ce544e168_iv -in Source/SupportingFiles/Credentials.swift.enc -out Source/SupportingFiles/Credentials.swift -d
   - for D in `find Tests -type d -maxdepth 1 -mindepth 1`; do echo "cp SupportingFiles/Credentials.swift $D/"; done
-
-after_success:
-  - bash <(curl -s https://codecov.io/bash)
-
 script:
   # build and test for iOS 10
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then set -o pipefail ; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then xcodebuild build -scheme "RestKit" -sdk "$IOS_SDK" -destination "$IOS10_DESTINATION" | xcpretty ; fi
```

Comment:
I actually tried adding the flag, and got this error: `xcodebuild: error: The flag -enableCodeCoverage is only supported when testing.`

When I think about it, this makes sense, since code coverage data is only collected when tests are run, to make sure that those tests are thorough.

We do, actually, get some information on code coverage values for Restkit, since all the other services use methods within Restkit. Restkit appears to be at about 78% right now.
![image](https://cloud.githubusercontent.com/assets/5377024/26743198/97b9819c-47a6-11e7-8d2c-2e57b15f9bae.png)